json.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import "fmt"
+
+type Peer struct {
+	Enode     string            `json:"enode"`
+	Id        peerId            `json:"id"`
```

Comment:
In go, shortcuts usually are uppercased. 
```suggestion
	ID        peerId            `json:"id"`
```
It's a convention of course, not a strict rule. So up to you in this project, consistency is more important IMO.

Reply:
Also, if `ID` is public I believe `peerId` also should be public.

Reply:
makes sense